android/app/src/androidTest/java/org/jitsi/meet/DetoxTest.java

```diff
@@ -0,0 +1,35 @@
+package org.jitsi.meet;
+
+import com.wix.detox.Detox;
+import com.wix.detox.config.DetoxConfig;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+import androidx.test.rule.ActivityTestRule;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class DetoxTest {
```

Comment:
Is this needed as some kind of pre-requisite for Detox to run?

Reply:
Yes, it's part of the Android setup.